tests/converter/rasa_converter_class_test.py

```diff
@@ -0,0 +1,9 @@
+"""This module tests the convertion tool that is used to convert a rasa or diaglogflow
+project into a mindmeld project"""
+
+from mindmeld.converter.rasa_converter import RasaConverter
+
+
+def test_rasa_converter(rasa_project_path, mindmeld_project_path):
+    converter = RasaConverter(rasa_project_path, mindmeld_project_path)
+    converter.convert_project()
```

Comment:
you can add a section to build project and test that the NLP component works correctly
```
nlp = NaturalLanguageProcessor(mindmeld_project_path)
nlp.build()
query1 = 'hi'
assert 'greeting' == nlp.process(query1)['intent']

Reply:
It's able to pass this test now.